try {
  hello();
} catch (error) {
  console.log(error);
}


try {
  another();
} catch (error) {
  console.log('Esto es un error!');
}